src/app/forms/Form.jsx

```diff
@@ -0,0 +1,124 @@
+      return;
+    }
+    toast(`✅ Submitted successfully!`);
+    console.log(object);
+  };
+  return (
+    <div className="w-full h-full overflow-scroll flex flex-col items-center font-poppins">
+      <div className="w-10/12 md:w-1/3 my-5">
+        <p className="text-xl bg-hackathon-green-300 font-semibold px-4 py-2 rounded-t-xl m-0">
+          {header}
+        </p>
+        <div className="rounded-b-xl bg-white p-3">
+          <Row className="flex justify-center p-0 m-0">
+            {fields.map((field, index) => (
+              <Col key={index} md={field.width}>
+                {field.input === "description" ? (
+                  field.texts.map((description, index) => (
+                    <p key={index}>{description}</p>
+                  ))
+                ) : field.input === "input" ? (
+                  <Input
+                    name={field.name}
+                    type={field.type}
+                    title={field.title}
+                    placeholder={field.placeholder}
+                    value={object[field.name]}
+                    maxLength={field.maxLength}
+                    user={object}
+                    setUser={setObject}
+                    editable={field.editable}
+                  />
+                ) : field.input === "select" ? (
+                  <Select
+                    title={field.title}
+                    options={field.options}
+                    field={field.field}
+                    placeholder={field.placeholder}
+                    user={object}
+                    setUser={setObject}
+                  />
+                ) : field.input === "checkboxes" ? (
+                  <>
+                    <p className="mb-1 mt-3">{field.text}</p>
+                    {Object.entries(object[field.field]).map(
+                      ([key, value], i) => (
+                        <Checkbox
+                          className="w-1/2"
+                          key={i}
+                          toggle={value}
+                          text={key}
+                          onClick={() =>
+                            setObject({
+                              ...object,
+                              [field.field]: {
+                                ...object[field.field],
+                                [key]: !value,
+                              },
+                            })
+                          }
+                          color="bg-hackathon-green-300"
+                        />
+                      )
+                    )}
+                  </>
+                ) : field.input === "radio" ? (
+                  <Radio
+                    text={field.text}
+                    options={field.options}
+                    field={field.field}
+                    user={object}
+                    setUser={setObject}
+                  />
+                ) : field.input === "textarea" ? (
```

Comment:
this could just be a bunch of && statements, which will prolly be easier to understand